### PR TITLE
AG-9234 - Add devFiles wildcard support.

### PR DESCRIPTION
**Pull request by alantreadway**
https://ag-grid.atlassian.net/browse/AG-9234

Adds `DEV_FILE_PATH_MAP` support for wildcards, which opens the door to chunked bundles for `ag-charts-community` and `ag-charts-enterprise` in the future if we decide to make that change.

### CODE REVIEW

File: nx.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
       "tsDeclarations",
       "sharedGlobals"
     ],
-    "sharedGlobals": ["{workspaceRoot}/rollup.*.js", "{workspaceRoot}/tsconfig.*.json"]
+    "sharedGlobals": ["{workspaceRoot}/rollup.*.cjs", "{workspaceRoot}/tsconfig.*.json"]
```
Comment (alantreadway) [inline nx.json:27]: NOTE: This is an unrelated fix to the dependencies for many targets.

File: packages/ag-charts-website/src/utils/pages.ts [hunk 2]
```diff
@@ -169,8 +164,7 @@ export const getBoilerPlateUrl = ({
  * Get dev files for local development
  */
 export function getDevFiles(): DevFileRoute[] {
-    const files = Object.keys(DEV_FILE_PATH_MAP).map((filePath) => {
-        const sourceFilePath = DEV_FILE_PATH_MAP[filePath];
+    const files = Object.entries(DEV_FILE_PATH_MAP).map(([filePath, sourceFilePath]) => {
```
Comment (taktran) [inline packages/ag-charts-website/src/utils/pages.ts:167]: 👍🏼 I'm sure I've not used `Object.entries` in other places where it would've been useful too

**Review by taktran — APPROVED** [review]
Nice addition 👍🏼 Good stuff!
